tests/Moq.Tests/MockRepositoryFixture.cs

```diff
@@ -76,25 +131,23 @@ public void ShouldVerifyVerifiables()
 		[Fact]
 		public void ShouldAggregateFailures()
 		{
-			try
-			{
-				var repository = new MockRepository(MockBehavior.Loose);
-				var foo = repository.Create<IFoo>();
-				var bar = repository.Create<IBar>();
+			var repository = new MockRepository(MockBehavior.Loose);
+			var foo = repository.Create<IFoo>();
+			var bar = repository.Create<IBar>();
 
-				foo.Setup(f => f.Do());
-				bar.Setup(b => b.Redo());
+			foo.Setup(f => f.Do());
+			bar.Setup(b => b.Redo());
 
-				repository.VerifyAll();
-			}
-			catch (MockException mex)
-			{
-				Expression<Action<IFoo>> fooExpect = f => f.Do();
-				Assert.Contains(fooExpect.ToString(), mex.Message);
+			var ex = Record.Exception(() => repository.VerifyAll());
 
-				Expression<Action<IBar>> barExpect = b => b.Redo();
-				Assert.Contains(barExpect.ToString(), mex.Message);
-			}
+			var expectedErrorMessage = $"The following setups on mock \'{foo}\' were not matched:\r\n" +
+										"IFoo f => f.Do()\r\n" +
+										"\r\n" +
+									   $"The following setups on mock \'{bar}\' were not matched:\r\n" +
+										"IBar b => b.Redo()\r\n";
```

Comment:
This and one other test currently fails on AppVeyor. I suspect this is due to your use of `"\r\n"` (CRLF) which isn't used on all system. Please try replacing it with `Environment.NewLine`.